resolvers/deno/sync.rs

```diff
@@ -43,7 +43,13 @@ mod inner {
   }
 
   impl<K: Eq + Hash, V, S: BuildHasher> MaybeDashMap<K, V, S> {
-    pub fn get<'a>(&'a self, key: &K) -> Option<Ref<'a, V>> {
+    pub fn get<'a, Q: Eq + Hash + ?Sized>(
+      &'a self,
+      key: &Q,
+    ) -> Option<Ref<'a, V>>
+    where
+      K: std::borrow::Borrow<Q>,
+    {
```

Comment:
Changed this because I found the crate wasn't compiling again when the `sync` feature isn't enabled. I think in the future we should add something to the CI to ensure all these crates compile without the `sync` feature and in Wasm, but atm it's not high priority.